packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -600,9 +610,9 @@ class _HeroFlight {
           end: 1.0,
         ),
       );
-
       if (manifest.fromHero != newManifest.toHero) {
-        manifest.fromHero.endFlight();
+        // Aborted animation should be placeheld.
```

Comment:
Do you have a test for this case?

Reply:
Also: why's the from hero placeheld?

Reply:
I don't think this will matter because the old heros are disposed, and it will not be testable because the elements are gone. I modify this just to be extra safe

Reply:
If it doesn't matter, I'd remove the comment. The comment makes it sound like that this is super-important.